lib/rubocop/cop/sorbet/binding_constants_without_type_alias.rb

```diff
@@ -69,7 +86,7 @@ def autocorrect(node)
           lambda do |corrector|
             corrector.replace(
               node.source_range,
-              "T.type_alias(#{node.source})"
+              "T.type_alias { #{node.source} }"
```

Comment:
Is this still not attempting to autocorrect?

Reply:
I kept the original behavior nesting 

```ruby
# bad
FooOrBar = T.any(Foo, Bar)
```

in:

```ruby
# good
FooOrBar = T.type_alias { T.any(Foo, Bar) }
```

What I removed is:

```ruby
if using_deprecated_type_alias_syntax?(node)
  corrector.replace(
    node.source_range,
    "T.type_alias { #{node.children[2].source} }"
  )
  return
end
```

Reply:
Oh, right. Sorry, I should have read the code better. Makes sense.